content/applications/general/users/access_rights.rst

```diff
@@ -1,67 +1,182 @@
 
-| When choosing the groups the user can have access under
-  :ref:`Access Rights <users/add-individual>`, details of the rules and inheritances of that group
-  are not shown, so this is when the menu *Groups* comes along. *Groups* are created to define rules
-  to models within an application.
-| Under *Users*, have a list of the current ones. The ones with administrative rights are shown
-  in black.
+.. danger::
+   Making changes to access rights can have a detrimental impact on the database. This includes
+   *impotent admin*, which means that no user in the database can make changes to the access rights.
+   For this reason, Odoo recommends contacting an Odoo Business Analyst, or our Support Team, before
+   making changes.
 
-.. image:: access_rights/groups-users.png
-   :align: center
-   :alt: View of a group’s form emphasizing the tab users in Odoo
+.. tip::
+   A user **must** have the specific *Administration* access rights set on their user profile, in
+   order to make changes on another user's settings for access rights.
+
+   To access this setting, navigate to :menuselection:`Settings app --> Manage users --> select a
+   user --> Access Rights tab --> Administration section --> Administration field`.
+
+   Once at the setting, an already existing administrator **must** change the setting in the
+   :guilabel:`Administration` field to :guilabel:`Access Rights`.
+
+   Once complete, click :guilabel:`Save` to save the changes, and implement the user as an
+   administrator.
+
+Users
+=====
 
-*Inherited* means that users added to this application group are automatically added to the
-following ones. In the example below, users who have access to the group *Administrator* of *Sales*
-also have access to *Website/Restricted Editor* and *Sales/User: All Documents*.
+The access rights for :ref:`individual users <users/add-individual>` are set when the user is added
+to the database, but they can be adjusted at any point in the user's profile.
 
-.. image:: access_rights/groups-inherited.png
+To make changes to a user's rights, click on the desired user to edit their profile.
+
+.. image:: access_rights/navigate-to-users-menu.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab inherited in Odoo
+   :alt: Users menu in the Users & Companies section of the Settings app of Odoo.
 
-.. important::
-   Remember to always test the settings being changed in order to ensure that they are being applied
-   to the needed and right users.
+On the user's profile page, in the :guilabel:`Access Rights` tab, scroll down to view the current
+permissions.
 
-The *Menus* tab is where you define which menus (models) the user can have access to.
+For each app, use the drop-down menu to select what level of permission this user should have. The
+options vary for each section, yet the most common are: *Blank/None*, *User: Own Documents*, *User:
+All Documents*, or *Administrator*.
```

Comment:
Updating these to guilabels since these are the most common drop-down menu options on this UI.
```suggestion
options vary for each section, yet the most common are: :guilabel:`Blank/None`, :guilabel:`User: Own
Documents`, :guilabel:`User: All Documents`, or :guilabel:`Administrator`.
```